scripts/akv/key_policy.json

```diff
@@ -0,0 +1,30 @@
+{
+    "issuerParameters": {
```

Comment:
Just a question, but who defines the json schema?

Reply:
Found that the cleanroom tool consumes it

Reply:
I don't think cleanroom consumes it unless I'm forgetting something?

I presume the policy is defined by AKV or some standard, it describes the member credentials